policy/multipolicyloader.go

```diff
@@ -96,7 +109,12 @@ func (l *MultiPolicyLoader) Load() (*Policy, Source, error) {
 // full root privileges.
 func ReadWithSudoScript(h *HomePolicyLoader, username string) ([]byte, error) {
 	// opkssh readhome ensures the file is not a symlink and has the permissions/ownership.
-	cmd := exec.Command("sudo", "-n", "/usr/local/bin/opkssh", "readhome", username)
+	opkBin, err := os.Executable()
+	if err != nil {
+		return nil, fmt.Errorf("error getting opkssh executable path: %w", err)
+	}
+	cmd := exec.Command("sudo", "-n", opkBin, "readhome", username)
```

Comment:
This is likely to break if the opkBin path is not `/usr/local/bin/` as we rely on an exact hardcoded path in the sudoers

https://github.com/openpubkey/opkssh/blob/5376c0aa91c477e08dae8cd16490e70d2a5d7e7d/scripts/install-linux.sh#L334-L339